app/controllers/api/v1/search_results_controller.rb

```diff
@@ -21,6 +19,23 @@ def create
       def upload_form_params
         params.permit(:search_engine, :csv_file)
       end
+
+      def process_results(search_results)
+        search_results.each do |search_result|
+          SearchKeywordJob.perform_later(search_result)
```

Comment:
It would be better to make this happen in the form object. Prefer to keep controllers lean so with minimal business logic. Controllers should just do routing to the right form objects and return responses.  In contrast, form objects can be as complex as required and can make as many operations as needed. They are also far easier to test than controllers.